tensorboard/webapp/metrics/store/metrics_reducers.ts

```diff
@@ -981,15 +981,28 @@ const reducer = createReducer(
   on(actions.rangeSelectionToggled, (state) => {
     const nextRangeSelectionEnabled = !state.rangeSelectionEnabled;
     let nextStepSelectorEnabled = state.stepSelectorEnabled;
+    let linkedTimeSelection = state.linkedTimeSelection;
 
     if (nextRangeSelectionEnabled) {
       nextStepSelectorEnabled = nextRangeSelectionEnabled;
+      if (!linkedTimeSelection) {
+        linkedTimeSelection = {
+          start: {step: state.stepMinMax.min},
+          end: {step: state.stepMinMax.max},
```

Comment:
Do we want to check the infinity/-infinity? It does not feel right to set that. If we set the selection to be infinity/-infinity, where do we rely on the correct steps of fobs? Clipping logic? (Will any warning icon pop up then?)

Reply:
I would rather not check for Infinity/-Infinity here. There isn't really an obvious value for this to fall back to otherwise and I think incorrectly clipped fobs (yes the clipping logic will come up here) seems better than the feature not working.